browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html

```diff
@@ -28,6 +48,58 @@
         </div>
       </div>
     </div>
+    <div class="settings-box">
+      <div class="flex">
+        <div class="label shields-primary-title">$i18n{defaultView}</div>
+        <settings-radio-group
+          pref="{{prefs.brave.advanced_view_default}}">
+          <controlled-radio-button class="shields-view-col" name="false">
+            <div class="flex-center">
+              <svg width="50" height="50" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+                <defs>
+                  <path d="M2 0h46a2 2 0 0 1 2 2v46a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V2a2 2 0 0 1 2-2z" id="simpleViewImgPath"/>
+                </defs>
+                <g fill="none" fill-rule="evenodd">
+                  <mask id="b" fill="#fff">
+                    <use xlink:href="#simpleViewImgPath"/>
+                  </mask>
+                  <path stroke="#CED0DB" d="M2 .5A1.5 1.5 0 0 0 .5 2v46A1.5 1.5 0 0 0 2 49.5h46a1.5 1.5 0 0 0 1.5-1.5V2A1.5 1.5 0 0 0 48 .5H2z"/>
+                  <ellipse fill="#CED0DB" mask="url(#b)" cx="25" cy="7.5" rx="49" ry="21.5"/>
+                </g>
+              </svg>
+              <label class="shields-view-label">$i18n{simpleView}</label>
+            </div>
+          </controlled-radio-button>
+          <div class="vertical-separator"></div>
+          <controlled-radio-button class="shields-view-col" name="true">
+            <div class="flex-center">
+              <svg width="50" height="80" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+                <defs>
+                  <path d="M2 0h46a2 2 0 0 1 2 2v76a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V2a2 2 0 0 1 2-2z" id="advViewImgPath" />
```

Comment:
nit: svg mask with ID being used, but perhaps not necessary and could cause issues

Reply:
updated